src/cmake/GeosxOptions.cmake

```diff
@@ -127,7 +127,7 @@ blt_append_custom_compiler_flag( FLAGS_VAR CMAKE_CXX_FLAGS
                                )
 
 blt_append_custom_compiler_flag( FLAGS_VAR CMAKE_CXX_FLAGS_DEBUG
-                                 GNU "-Wno-unused-parameter -Wno-unused-variable"
+                                 GNU "-Wno-unused-parameter -Wno-unused-variable -Wno-dangling-reference"
```

Comment:
What was this required for? i.e. where is the dangling reference that this avoids?